jupyterhub_moss/form/option_form.js

```diff
@@ -86,15 +89,88 @@ function updatePartitionLimits() {
   });
 }
 
+function storeConfigToLocalStorage() {
+  const advancedDiv = document.getElementById("menu1");
+  const runtimeSelect = document.getElementById('runtime_simple');
+
+  // Retrieve form fields to store
+  const fieldNames = ['partition', 'nprocs', 'ngpus', 'runtime', 'jupyterlab',
+                      'exclusive', 'reservation', 'nnodes', 'ntasks'];
+  var fields = {}
```

Comment:
you can use `let` here

Reply:
Got it, object is mutable and ref is const